### PR TITLE
CEC-685:Runtime Implementation

### PR DESCRIPTION
**Pull request by Shubhangi-cs**

### CODE REVIEW

File: src/main/java/com/google/cloud/datafusion/plugin/successfactors/source/input/SuccessFactorsInputFormat.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
```
Comment (sourabhrawat345) [inline src/main/java/com/google/cloud/datafusion/plugin/successfactors/source/input/SuccessFactorsInputFormat.java]: update to 2022 in other files also

File: src/main/java/com/google/cloud/datafusion/plugin/successfactors/source/input/SuccessFactorsInputFormat.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+/**
+ * This {@code SuccessFactorsInputFormat} defines the InputFormat implementation for SuccessFactors.
+ */
+public class SuccessFactorsInputFormat extends InputFormat<LongWritable, StructuredRecord> {
+
+  public static final String SUCCESSFACTORS_PLUGIN_PROPERTIES = "successFactorsPluginProperties";
+  public static final String PARTITIONS_PROPERTY = "partitionProperty";
+  public static final String ENCODED_ENTITY_METADATA_STRING = "encodedMetadataString";
+
+
```
Comment (sourabhrawat345) [inline src/main/java/com/google/cloud/datafusion/plugin/successfactors/source/input/SuccessFactorsInputFormat.java]: remove empty lines

**Review by sourabhrawat345 — APPROVED** [review]

File: src/main/java/com/google/cloud/datafusion/plugin/successfactors/source/input/SuccessFactorsInputFormat.java (hunk 1 above)
Comment (sgarg-CS) [reply src/main/java/com/google/cloud/datafusion/plugin/successfactors/source/input/SuccessFactorsInputFormat.java]: Updated.

File: src/main/java/com/google/cloud/datafusion/plugin/successfactors/source/input/SuccessFactorsInputFormat.java (hunk 2 above)
Comment (sgarg-CS) [reply src/main/java/com/google/cloud/datafusion/plugin/successfactors/source/input/SuccessFactorsInputFormat.java]: Removed.

**Pull request merged**